2025-05-30 11:31:03,911 INFO    MainThread:16806 [wandb_setup.py:_flush():70] Current SDK version is 0.19.11
2025-05-30 11:31:03,911 INFO    MainThread:16806 [wandb_setup.py:_flush():70] Configure stats pid to 16806
2025-05-30 11:31:03,911 INFO    MainThread:16806 [wandb_setup.py:_flush():70] Loading settings from /Users/juliankleutgens/.config/wandb/settings
2025-05-30 11:31:03,911 INFO    MainThread:16806 [wandb_setup.py:_flush():70] Loading settings from /Users/juliankleutgens/PycharmProjects/ratio_estimation_synthetic/wandb/settings
2025-05-30 11:31:03,912 INFO    MainThread:16806 [wandb_setup.py:_flush():70] Loading settings from environment variables
2025-05-30 11:31:03,912 INFO    MainThread:16806 [wandb_init.py:setup_run_log_directory():724] Logging user logs to /Users/juliankleutgens/PycharmProjects/ratio_estimation_synthetic/wandb/run-20250530_113103-ho9ds843/logs/debug.log
2025-05-30 11:31:03,912 INFO    MainThread:16806 [wandb_init.py:setup_run_log_directory():725] Logging internal logs to /Users/juliankleutgens/PycharmProjects/ratio_estimation_synthetic/wandb/run-20250530_113103-ho9ds843/logs/debug-internal.log
2025-05-30 11:31:03,912 INFO    MainThread:16806 [wandb_init.py:init():852] calling init triggers
2025-05-30 11:31:03,912 INFO    MainThread:16806 [wandb_init.py:init():857] wandb.init called with sweep_config: {}
config: {'data_type': 'point_forms', 'unguided_sampling': False, 'skip_ratio_estimator_on_clean_data': True, 'test_run': False, 'seed': 100, 'len_seqs': 10, 'vocab_size': 5, 'number_of_source_samples': 10000, 'number_of_target_samples': 1000, 'dimension_gaussian': 2, 'num_mixtures': 2, 'grid_size': 1.5, 'quantization_step': 0.01, 'source_form': 'pinwheel', 'target_form': 'swissroll', 'source_dist': 'uniform', 'target_dist': 'gaussian', 'distance': 0, 'grid_size_discrete': 10, 'type_of_T_matrix': 'diagonal', 'extra_target_tokens': 1, 'dirichlet_alpha': 4.0, 'batch_size': 256, 'device': device(type='mps'), 'train_classifier_epochs': 30, 'train_ratio_estimator_epochs': 30, 'train_denoiser_epochs': 30, 'denoiser_dim': 256, 'denoiser_heads': 4, 'denoiser_layers': 4, 'sample_batch_size': 1024, 'sample_gammas': [0, 0.5, 1, 2], 'sample_size': 16384, 'denoising_steps': 20, 'use_plg': True, 'eta': 1.0, '_wandb': {}}
2025-05-30 11:31:03,912 INFO    MainThread:16806 [wandb_init.py:init():893] starting backend
2025-05-30 11:31:03,912 INFO    MainThread:16806 [wandb_init.py:init():897] sending inform_init request
2025-05-30 11:31:03,942 INFO    MainThread:16806 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2025-05-30 11:31:03,943 INFO    MainThread:16806 [wandb_init.py:init():907] backend started and connected
2025-05-30 11:31:03,947 INFO    MainThread:16806 [wandb_init.py:init():1005] updated telemetry
2025-05-30 11:31:03,947 INFO    MainThread:16806 [wandb_init.py:init():1029] communicating run to backend with 90.0 second timeout
2025-05-30 11:31:04,338 INFO    MainThread:16806 [wandb_init.py:init():1104] starting run threads in backend
2025-05-30 11:31:04,422 INFO    MainThread:16806 [wandb_run.py:_console_start():2573] atexit reg
2025-05-30 11:31:04,423 INFO    MainThread:16806 [wandb_run.py:_redirect():2421] redirect: wrap_raw
2025-05-30 11:31:04,423 INFO    MainThread:16806 [wandb_run.py:_redirect():2490] Wrapping output streams.
2025-05-30 11:31:04,423 INFO    MainThread:16806 [wandb_run.py:_redirect():2513] Redirects installed.
2025-05-30 11:31:04,426 INFO    MainThread:16806 [wandb_init.py:init():1150] run started, returning control to user process
2025-05-30 11:31:04,426 INFO    MainThread:16806 [wandb_run.py:_config_callback():1436] config_cb None None {'data_type': 'point_forms', 'unguided_sampling': False, 'skip_ratio_estimator_on_clean_data': True, 'test_run': False, 'seed': 100, 'len_seqs': 10, 'vocab_size': 5, 'number_of_source_samples': 10000, 'number_of_target_samples': 1000, 'dimension_gaussian': 2, 'num_mixtures': 2, 'grid_size': 1.5, 'quantization_step': 0.01, 'source_form': 'pinwheel', 'target_form': 'swissroll', 'source_dist': 'uniform', 'target_dist': 'gaussian', 'distance': 0, 'grid_size_discrete': 10, 'type_of_T_matrix': 'diagonal', 'extra_target_tokens': 1, 'dirichlet_alpha': 4.0, 'batch_size': 256, 'device': 'mps', 'train_classifier_epochs': 30, 'train_ratio_estimator_epochs': 30, 'train_denoiser_epochs': 30, 'denoiser_dim': 256, 'denoiser_heads': 4, 'denoiser_layers': 4, 'sample_batch_size': 1024, 'sample_gammas': [0, 0.5, 1, 2], 'sample_size': 16384, 'denoising_steps': 20, 'use_plg': True, 'eta': 1.0}
2025-05-30 11:32:01,815 WARNING MsgRouterThr:16806 [router.py:message_loop():63] [no run ID] message_loop has been closed
Traceback (most recent call last):
  File "/Users/juliankleutgens/opt/anaconda3/envs/tgdp/lib/python3.9/site-packages/wandb/sdk/interface/router_sock.py", line 27, in _read_message
    return self._sock_client.read_server_response(timeout=1)
  File "/Users/juliankleutgens/opt/anaconda3/envs/tgdp/lib/python3.9/site-packages/wandb/sdk/lib/sock_client.py", line 235, in read_server_response
    data = self._read_packet_bytes(timeout=timeout)
  File "/Users/juliankleutgens/opt/anaconda3/envs/tgdp/lib/python3.9/site-packages/wandb/sdk/lib/sock_client.py", line 220, in _read_packet_bytes
    raise SockClientClosedError
wandb.sdk.lib.sock_client.SockClientClosedError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/juliankleutgens/opt/anaconda3/envs/tgdp/lib/python3.9/site-packages/wandb/sdk/interface/router.py", line 56, in message_loop
    msg = self._read_message()
  File "/Users/juliankleutgens/opt/anaconda3/envs/tgdp/lib/python3.9/site-packages/wandb/sdk/interface/router_sock.py", line 29, in _read_message
    raise MessageRouterClosedError from e
wandb.sdk.interface.router.MessageRouterClosedError
2025-05-30 11:32:01,816 INFO    MsgRouterThr:16806 [mailbox.py:close():129] [no run ID] Closing mailbox, abandoning 1 handles.
